pom.xml

```diff
@@ -217,7 +220,7 @@
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/parameterized-trigger-plugin</gitHubRepo>
     <java.level>8</java.level>
-    <jenkins.version>2.270</jenkins.version>
+    <jenkins.version>2.289.1</jenkins.version>
```

Comment:
as recommended here https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/
can I change to `later releases within an LTS line`
thanks

Reply:
The current PR matches the currently documented advice, but FWIW there is no real consensus here. See https://github.com/jenkins-infra/jenkins.io/pull/4876

Reply:
Thanks for your comments! 
Lmk if we mutually decide to change `jenkins.version` be 2.289.3. I'm keeping it as 2.289.1 for now, unless Olivier has something to add.